sympy/core/numbers.py

```diff
@@ -25,6 +25,7 @@
     fnan as _mpf_nan, fzero as _mpf_zero, _normalize as mpf_normalize,
     prec_to_dps)
 from sympy.utilities.misc import debug, filldedent
+from sympy.mpmath.libmp.backend import MPZ
```

Comment:
This needs to be just `mpmath.libmp.backend`. Mpmath isn't included with SymPy any more.